src/ResourceManager/Network/Commands.Network/help/Add-AzureRmLoadBalancerRuleConfig.md

```diff
@@ -90,6 +92,21 @@ Accept pipeline input: False
 Accept wildcard characters: False
 ```
 
+### -DisableOutboundSNAT
+DisableOutboundSNAT
```

Comment:
@EvgenyAgafonchikov would you mind updating this description to be more meaningful to the user?

Reply:
@DeepakRajendranMsft could you please contact owners of this update to ask them are they good with renaming of the parameter and what name would be applicable there from their point of view?

Reply:
Replaced description with one from rest api spes